tests/src/whisk/core/apigw/actions/test/ApiGwRoutemgmtActionTests.scala

```diff
@@ -0,0 +1,203 @@
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "action is missing the backendMethod field",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{}}")),
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "action is missing the backendUrl field",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{\"backendMethod\":\"post\"}}")),
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "action is missing the namespace field",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{\"backendMethod\":\"post\",\"backendUrl\":\"URL\"}}")),
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "action is missing the name field",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{\"backendMethod\":\"post\",\"backendUrl\":\"URL\",\"namespace\":\"_\"}}")),
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "action is missing the authkey field",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{\"backendMethod\":\"post\",\"backendUrl\":\"URL\",\"namespace\":\"_\",\"name\":\"N\"}}")),
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "swagger and gatewayBasePath are mutually exclusive and cannot be specified together",
+              Seq("-p", "apidoc", "{\"namespace\":\"_\",\"gatewayBasePath\":\"/ApiGwRoutemgmtActionTests_bp\",\"gatewayPath\":\"ApiGwRoutemgmtActionTests_rp\",\"gatewayMethod\":\"get\",\"action\":{\"backendMethod\":\"post\",\"backendUrl\":\"URL\",\"namespace\":\"_\",\"name\":\"N\",\"authkey\":\"XXXX\"},\"swagger\":{}}")),
+
+            ("/whisk.system/routemgmt/createRoute", ANY_ERROR_EXIT, "apidoc field cannot be parsed. Ensure it is valid JSON",
+              Seq("-p", "namespace", "_", "-p", "apidoc", "{1:[}}}"))
+        )
+
+        invalidArgs foreach {
+            case (action:String, exitcode:Int, errmsg:String, params:Seq[String]) =>
+                var cmd: Seq[String] = Seq("action",
+                    "invoke",
+                    action,
+                    "-i", "-b", "-r",
+                    "--apihost", wskprops.apihost,
+                    "--auth", whisksystemAuthId+":"+whisksystemAuthPwd
+                ) ++ params
```

Comment:
I believe the `wsk.action.invoke` hardwired the use of the whisk.properties auth key (auth.guest).  When invoking these actions, I needed to use the /whisk.system's auth key.